src/mavo.js

```diff
@@ -177,7 +177,18 @@ var _ = self.Mavo = $.Class({
 
 		this.permissions.can("login", () => {
 			// We also support a URL param to trigger login, in case the user doesn't want visible login UI
-			if (Mavo.Functions.url("login") !== null && this.index == 1 || Mavo.Functions.url(this.id + "-login") !== null) {
+			let loginUrlParam = null;
```

Comment:
No need to initialize to `null`, JS variables are automatically initialized to `undefined`, so you can just compare with `undefined` later.